src/js/features/beacon/beaconQuery.js

```diff
@@ -0,0 +1,37 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
```

Comment:
this file doesn't follow the naming scheme followed by other public features

Reply:
as above